platform/core.multitabs/src/org/netbeans/core/multitabs/impl/ProjectColorTabDecorator.java

```diff
@@ -161,8 +161,16 @@ public void paintAfter( TabData tab, Graphics g, Rectangle tabRect, boolean isSe
         }
         g.setColor( c );
         Rectangle rect = new Rectangle( tabRect );
-        rect.y += rect.height - 3;
```

Comment:
Realizing this code was already there... what's going on here? Is tabRect the entire tab to start with?

Reply:
Sorry, stray comment... I got it.